### DISCUSSION TITLE
SVG More Option

### DISCUSSION THREAD

**Discussion by matrixmorfeus** [arkypita/LaserGRBL#1607] [Ideas]
Hi, is possible to have more option for SVG file? 
Infill, line+infill... invert infill!!! Is Possible???

Is possible to have a filling for path???

**Comment by alastaira** [comment]
This would seem like a really useful feature - essentially, to mirror the functionality of the options available when "vectorising" a JPG/PNG file, but for files that were already vector in the first place.

**Comment by arkypita** [comment]
An SVG file is made with outlines only, it is a limitation of the file format itself. In the future, perhaps LaserGRBL could carry out some filling treatments, but this option is not currently foreseen.

I suggest you do line fills directly in your vector editor, this way you have maximum control over how the filling is done.
Here is some resources:

https://www.youtube.com/watch?v=wYzMujtnkfM

https://www.youtube.com/watch?v=zn3vQAD_13Q

https://www.youtube.com/watch?v=ygMbgMNgunc

https://wiki.evilmadscientist.com/Hatch_fill

http://tavmjong.free.fr/INKSCAPE/MANUAL/html/Paths-LivePathEffects-Hatches.html

https://wiki.evilmadscientist.com/Creating_filled_regions

However, I do not exclude that one day this can also be done directly in LaserGRBL, but for now I cannot implement this functionality quickly.

**Comment by arkypita** [comment]
PS. this is not an issue, so please post under "discussion" session.
